.travis/test_06_script_a.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2018 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
```

Comment:
15303, this is too much, shouldn't just add this whole file

Reply:
bitcoin#15303 does rename .travis/test_06_script.sh → .travis/test_06_script_a.sh  I will partially backport Merge #13863: for this file (as #13863 mentioned DNM with comments as "partial (everything relevant was merged)")